static/modal_style.css

```diff
@@ -0,0 +1,294 @@
+/* Modals */
+
+.modal-1 {
+  width: 100%;
+  height: 850px;
+  background-color: #3c3a39;
+  color: #fff5e1;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Roboto', sans-serif;
+  border: 0;
+}
```

Comment:
[OPTIONAL] Good job implementing the changes requested by the previous Reviewer. But if you carefully look at the pop-up in the desktop view of your project the Multi Post Stories is not supposed to be visible. it would be nice if you addressed this issue by removing Multi Post Stories text to match the pop-up design for your desktop version.


This is your design:

 
![Screenshot 2023-06-29 at 2 07 25 PM](https://github.com/wacoo/Portfolio-Project/assets/79506105/d3e9a44c-a40e-4c32-9e61-c52d199f27a2)

Please follow this design:




![Screenshot 2023-06-29 at 2 07 12 PM](https://github.com/wacoo/Portfolio-Project/assets/79506105/9c2dae1f-dc6a-45a0-9030-0330c4f0be3c)